src/components/DataSelector/RendersOptionSelect/index.tsx

```diff
@@ -2,25 +2,19 @@ import { Alert, AlertDescription, AlertIcon, AlertTitle } from "@chakra-ui/alert
 import { Button, FormControl, FormErrorMessage, Radio, RadioGroup, Stack } from "@chakra-ui/react";
 import { useForm, Controller, SubmitHandler } from "react-hook-form"
 import { SelectProps } from "./types";
-import { durationToMs } from "../../../utils";
+import { getMostRecentUTC } from "../../../utils";
 
 type FormValues = {
   renderOption: string;
   datetime: string;
 }
 
 function VariablesSelect({ collection, addLayer }: SelectProps) {
-  const { stac, datetime_range } = collection;
-  const cubeVariables = stac['cube:variables'];
-  const variableOptions = cubeVariables ? Object.keys(cubeVariables) : null;
+  const { stac } = collection;
   const renderOptions = Object.keys(stac.renders);
-  // layerOptions will be keys of cube:variables (variableOptions) if they exist,
-  // otherwise the options are the keys of the renders object (renderOptions).
-  const layerOptions = variableOptions || renderOptions;
-
-  // Simplified time determination logic
-  const timeInfo = stac['cube:dimensions']?.time || stac.extent.temporal;
-  const timeMin = (stac['cube:dimensions'] ? timeInfo.extent[0] : timeInfo.interval[0][0]) || '1970-01-01T00:00:00Z';
+  const temporal = stac.extent.temporal;
+  const lastTemporalExtent = temporal.interval[0][1];
+  const maxDatetimeStr = lastTemporalExtent ? lastTemporalExtent : getMostRecentUTC().toISOString();
```

Comment:
`getMostRecentUTC().toISOString()` needs to change to most recent data available, since data is not being continually ingested yet